### PR TITLE
Add Cytomining Ecosystem publication data gathering and analysis

### PR DESCRIPTION
**Pull request by d33bs**
This PR adds Cytomining Ecosystem publication data gathering and analysis. Candidly, this just scratches the surface in terms of bibliometrics! I thought this might be enough to provide some insight on these projects for the time being.

Thanks in advance for any thoughts and feedback you may have!

### CODE REVIEW

File: notebooks/cytomining-ecosystem/gather-targeted-publication-mentions.ipynb [hunk 1]
```diff
@@ -711,17 +706,17 @@
        "</div>"
       ],
       "text/plain": [
-       "  Project Name  Date Created Year  Date Created Year_2  google_scholar_count  \\\n",
-       "0  pycytominer               2019                 2019                    20   \n",
-       "1    CytoSnake               2022                 2022                     0   \n",
-       "2    CytoTable               2022                 2022                     0   \n",
-       "3   IDR_stream               2022                 2022                     0   \n",
+       "  Project Name  Date Created Year  google_scholar_count  bioarxiv_count  \\\n",
```
Comment (gwaybio) [inline notebooks/cytomining-ecosystem/gather-targeted-publication-mentions.ipynb]: typo in biorxiv

File: notebooks/cytomining-ecosystem/gather-targeted-publication-mentions.ipynb [hunk 2]
```diff
@@ -711,17 +706,17 @@
...
+       "0  pycytominer               2019                    20              13   \n",
+       "1    CytoSnake               2022                     0               0   \n",
+       "2    CytoTable               2022                     0               0   \n",
+       "3   IDR_stream               2022                     0               0   \n",
```
Comment (gwaybio) [inline notebooks/cytomining-ecosystem/gather-targeted-publication-mentions.ipynb]: Let's remove IDR_stream (from this and the other landscape analysis)

**Review by gwaybio — APPROVED** [review]
Love to see this!

File: notebooks/cytomining-ecosystem/gather-targeted-publication-mentions.ipynb (hunk 1 above)
Comment (d33bs) [reply notebooks/cytomining-ecosystem/gather-targeted-publication-mentions.ipynb]: Thanks! Will get this updated.

File: notebooks/cytomining-ecosystem/gather-targeted-publication-mentions.ipynb (hunk 2 above)
Comment (d33bs) [reply notebooks/cytomining-ecosystem/gather-targeted-publication-mentions.ipynb]: Got it, thanks - will do. I'll keep the focus isolated for now, but we likely will need to re-run the full workflow as a separate PR.

**Comment by d33bs** [comment]
Thanks @gwaybio !

**Pull request merged**